### ISSUE TITLE
Transform timestamp into ISO8601 dates for registered claims.

### ISSUE THREAD

**Issue by Delapouite** [mike-engel/jwt-cli#103]
Hello

First, let me thank you for this great tool!

I have a feature suggestion regarding the output of the `decode` command.
The registered claims `iat`, `exp` and `nbf` are decoded as timestamps which is fine.

But sometimes, it would be easier to see these dates formatted for human consumption.

Current output:
```json
  "header": {
    "alg": "HS512"
  },
  "payload": {
    "exp": 1606760228,
    "iat": 1606751228
  }
}
```
So, would a `jwt decode --json --iso8601` would make sense to produce something like?:

```json
  "header": {
    "alg": "HS512"
  },
  "payload": {
    "exp": "2020-11-30T18:17:08Z",
    "iat": "2020-11-30T15:47:08Z"
  }
}
```

**Comment by mike-engel** [comment]
Hi @Delapouite. Thanks for filing this issue!

My first idea was to follow what tools like `du` do and use `-h` for human readable output, but that's already taken by the help command.

I think that a new `--iso8601` flag for `decode` only (for now) makes sense to me. Would you want to put the PR together? I'm not sure how much experience you have with rust.

**Comment by mike-engel** [comment]
Released with 4.0.0
